### DISCUSSION TITLE
How to download all photos in an album?

### DISCUSSION THREAD

**Discussion by mooserita** [kevinzg/facebook-scraper#1064] [Q&A]
Hello,
The readme has a reference to image albums being downloadable with this scraper (it's crossed out in the to-do list at the bottom), but it does not have an example on how to do that.  Is there a help file or more instructions on that?

Specifically, I'm trying to download a friend's photo albums since they have lost access to their account and asked me to try and salvage their photos if possible.  We're still friends on FB so I can see their photo albums.

Any help would be greatly appreciated!

**Comment by RameezRaja1** [comment]
Hello,

Downloading image albums can be tricky if not clearly documented. Since the feature is crossed out, it might not be fully implemented yet.

For help with growing your social media presence and managing followers, you might find (https://topfollowerapkss.com/) useful. They offer great tools and tips.

As for your current need, you can try these steps:

Check the Scraper’s GitHub Issues/Discussions: Sometimes, users share additional instructions there.
Contact the Developer: Reach out to the scraper’s developer for detailed guidance.
Alternative Tools: Look for other tools that specifically mention downloading Facebook photo albums.
Good luck!
